createst.py

```diff
@@ -355,6 +357,8 @@ def parse_args():
                         help="Stricly validate checksum")
     parser.add_argument("--min-version", default=None, metavar="<min-version>",
                         help="Adds a global minimum required version")
+    parser.add_argument("--c", metavar="<path-to-suricata.yaml>",
```

Comment:
double dash c `--c` seems a bit of an anti pattern to me. Generally we reserve single letter options for single dash. Maybe we should use `--cfg`? or `--suricata-yaml`?